spec/workers/send_new_cycle_has_started_email_to_candidates_worker_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe SendNewCycleHasStartedEmailToCandidatesWorker, sidekiq: true do
+  describe '#perform' do
+    it 'sends emails to candidates who have unsuccessful or unsubmitted applicatonis from the previous cycle' do
+      candidate_1 = create(:candidate)
+      candidate_2 = create(:candidate)
+      unsubmitted_application_choice = create(:application_choice, :application_not_sent)
```

Comment:
Is this use of the `application_not_sent` factory trait correct or should I  be using an application form with `submitted_at` equal to  `nil`

Reply:
Um i think we probably shouldn't use that trait as it's a deprecated state